README.md

```diff
@@ -1,5 +1,6 @@
-# Wikimedia Commons Android app [![Build status](https://api.travis-ci.org/commons-app/apps-android-commons.svg?branch=master)](https://travis-ci.org/commons-app/apps-android-commons)
-
+# Wikimedia Commons Android app
+[![Build status](https://api.travis-ci.org/commons-app/apps-android-commons.svg?branch=master)](https://travis-ci.org/commons-app/apps-android-commons)
```

Comment:
@madhurgupta10 Just a tip, as mentioned in the [Make separate commits for logically separate changes](https://github.com/commons-app/apps-android-commons/wiki/Contributing-Guidelines#make-separate-commits-for-logically-separate-changes) sub-section of the [Contributing Guidelines](https://github.com/commons-app/apps-android-commons/wiki/Contributing-Guidelines), it would have been nice if you had made separate commits for adding the Appetize badges and the Travis-CI build badge and made it clear in the PR description.

No one would expect a PR that says that it's adding the appetize badge to **also** add the Travis-CI badge. So, it would be nice if you could make it more clear.